src/plugins/copyCode.js

```diff
@@ -0,0 +1,77 @@
+const cheerio = module.parent.require('cheerio');
+
+const {
+  COPIED_TO_CLIPBOARD,
+  COPY_ICON,
+  COPY_TO_CLIPBOARD,
+} = require('../constants');
+
+function getButtonHTML() {
+  const html = `<button class="copy-btn-icon copy-btn" type="button">
+    <div class="copy-btn-body">
+    ${COPY_ICON}
+    <strong class="copy-btn-label">${COPY_TO_CLIPBOARD}</strong>
+    </div>
+    </button>`;
+  return html;
+}
+
+function getCopyCodeBlockScript() {
+  return `<script>
+    document.onclick = ((event) => {
```

Comment:
It feels really strange to have an `onclick` handler for the entire document, then check to see if it's hitting the button. Can we just add an `onclick` handler to the button directly instead?

https://www.w3schools.com/jsref/event_onclick.asp

Reply:
Oh yeah, you're right 😅 Made the change 👍